game/scripts/vscripts/modules/illusions/illusions.lua

```diff
@@ -82,17 +82,48 @@ end
 
 function Illusions:_copyAppearance(unit, illusion)
 	illusion:SetModelScale(unit:GetModelScale())
-	if unit:GetModelName() ~= illusion:GetModelName() then
-		illusion.ModelOverride = unit:GetModelName()
-		illusion:SetModel(illusion.ModelOverride)
-		illusion:SetOriginalModel(illusion.ModelOverride)
-	end
 	local rc = unit:GetRenderColor()
 	if rc ~= Vector(255, 255, 255) then
 		illusion:SetRenderColor(rc.x, rc.y, rc.z)
 	end
 end
 
+local COPYABLE_BUFFS = {
+	modifier_alchemist_chemical_rage = 1,
+	modifier_arc_warden_tempest_double = 1,
+	modifier_troll_warlord_berserkers_rage = 1,
+	modifier_dragon_knight_dragon_form = 1,
+	modifier_lone_druid_true_form = 1,
+	modifier_terrorblade_metamorphosis_transform = 1,
+	modifier_invoker_quas_instance = 1,
+	modifier_invoker_wex_instance = 1,
+	modifier_invoker_exort_instance = 1,
+	modifier_undying_flesh_golem = 1,
+	modifier_lycan_shapeshift = 1,
+	modifier_apocalypse_apocalypse = 1,
+	modifier_sai_release_of_forge = 1,
+}
+
+local illusion_getElaspedTime = function(self)
+	return self:OldGetElaspedTime() + self.SourceElaspedTime
+end
+
+function Illusions:_copyBuffs(unit, illusion)
+	for _, v in pairs(unit:FindAllModifiers()) do
+		local buffName = v:GetName()
+		local buffAbility = v:GetAbility()
+		local buff_copyStatus = COPYABLE_BUFFS[buffName]
+		if buff_copyStatus == 1 then
```

Comment:
Now, that there is only one copy type, change it to `if COPYABLE_BUFFS[buffName] then` and replace COPYABLE_BUFFS values with `true`